app/views/external_users/claims/hardship_fee/_fields.html.haml

```diff
@@ -1,33 +1,39 @@
 #hardship-fee
   = f.fields_for :hardship_fee do |hf|
     - fee = present(hf.object)
-    .form-section.fx-fee-group.hardship-fee-group.js-block.fx-do-init{data:{"type": "hardshipFees", autovat: @claim.apply_vat? ? "true" : "false", "block-type": "FeeBlock"}}
-      = hf.hidden_field :fee_type_id, value: fee.fee_type.id, class: 'js-fee-type'
+    .fx-fee-group.hardship-fee-group.js-block.fx-do-init{data:{"type": "hardshipFees", autovat: @claim.apply_vat? ? "true" : "false", "block-type": "FeeBlock"}}
 
-      .js-fee-calculator-effector
-        = hf.adp_text_field :quantity,
-          label: t('.quantity'),
-          hint_text: t('.quantity_hint'),
-          input_classes:'quantity fee-quantity js-fee-calculator-ppe form-control-1-4',
-          input_type: 'number',
-          value: fee.quantity,
-          errors: @error_presenter,
-          error_key:'hardship_fee.quantity'
+      = hf.govuk_text_field :fee_type_id,
+        class: 'js-fee-type',
+        form_group: { classes: 'govuk-visually-hidden' },
+        label: { hidden: true },
+        type: 'hidden',
+        value: fee.fee_type.id
 
-      .js-graduated-price-effectee
-        .calculated-grad-fee
-        = hf.adp_text_field :amount,
-          label: t('.amount'),
-          input_classes: 'fee-amount form-control-1-4 form-input-denote__input total',
-          input_type: 'currency',
-          value: number_with_precision(hf.object.amount, precision: 2),
-          id: 'hardship_fee_attributes_amount',
-          errors: @error_presenter,
-          error_key:'hardship_fee.amount'
+      = hf.govuk_number_field :quantity,
+        class:'quantity fee-quantity js-fee-calculator-ppe form-control-1-4',
```

Comment:
CSS class `form-control-1-4` is redundant

Reply:
Done in latest commit